src/components/follow-them/follow-them.js

```diff
@@ -1,9 +1,83 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import '../../styles.css';
 import avatar from '../../assets/defaultImg.png';
 import './follow-them.css';
+import { useAuth } from '../../context/authContext';
+import { doc, updateDoc, arrayUnion, arrayRemove } from '@firebase/firestore';
+import { db } from '../../firebase';
+
+export default function FollowThem({ userObj }) {
+    const { fullName, username, following, uid, id, followers } = userObj;
+    const { user, setUser } = useAuth();
+
+    const addToFollowing = async () => {
+        try {
+            console.log(uid);
+            console.log(user.following[2]);
+            console.log(user.following[2] === uid)
```

Comment:
If not required, you can *remove those console.logs*

Reply:
Done [2cac699](https://github.com/shivanigangadharan/social-media/pull/8/commits/2cac699d200222db146bddaf9481099ca56852be)